### DISCUSSION TITLE
Remove points added dynamically

### DISCUSSION THREAD

**Discussion by zhengliuX** [pyvista/pyvista#1803] [Q&A] [answered]
Hi, I'm trying to build an app with pyvista, and I want to update the plotter with some buttons. 
For example, some points have been already added to the plotter(using pyvistaqt), and the user wants to disable/enable some of the points' visualization with a button. How do I dynamically control, or remove some of the points added in the plotter?
I wonder is there an API that could do plotter.remove(xyz_coordniates)

**Comment by banesullivan** [comment, marked as answer]
After adding a mesh through the plotter's `add_mesh` method, an actor is returned. You can implement a callback on a button to set the visibility of that actor.

Here is an example where this previously came up:

https://github.com/pyvista/pyvista-support/issues/182#issuecomment-645482993
